arangod/Cluster/ClusterMethods.cpp

```diff
@@ -1351,6 +1351,41 @@ futures::Future<OperationResult> countOnCoordinator(
   return futures::collectAll(std::move(futures)).thenValue(std::move(cb));
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// @brief gets the metrics from DBServers
+////////////////////////////////////////////////////////////////////////////////
+
+futures::Future<metrics::RawDBServers> metricsOnCoordinator(
+    NetworkFeature& network, ClusterFeature& cluster) {
+  auto* pool = network.pool();
+  auto serverIds = cluster.clusterInfo().getCurrentDBServers();
+
+  std::vector<Future<network::Response>> futures;
+  futures.reserve(serverIds.size());
+  for (auto const& id : serverIds) {
+    network::Headers headers;
+    headers.emplace(StaticStrings::Accept,
+                    StaticStrings::MimeTypeJsonNoEncoding);
+    futures.push_back(network::sendRequest(
+        pool, "server:" + id, fuerte::RestVerb::Get, "/_admin/metrics/v2", {},
+        network::RequestOptions{}.param("json", "y").param("coordinator", "y"),
```

Comment:
Can we turn the "y" into "true" here? That would be consistent with our other APIs that handle boolean inputs.

Reply:
Ok, for consistency